issue_template.md

```diff
@@ -0,0 +1,31 @@
+#### brief description
```

Comment:
Could you add the sentence.
"""
Please understand that this is not a forum but an issue tracker, and read the following article to understand why: [Why you must not ask questions on Github issues](https://medium.com/@methane/why-you-must-not-ask-questions-on-github-issues-51d741d83fde)

Invalid issues while be closed without any comment.
"""

Reply:
Added a section on top 'A note about questions vs. issues' on top of the whole template.